src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -62,14 +62,69 @@
-                "Note that for any pair of variants to compare, only the alleles for the samples under interrogation are considered " +
-                "and MNP, Symbolic, and Mixed classes of variants are not included.",
-        usageShort = "Calculates the concordance between genotype data for two samples in two different VCFs",
+        usage = GenotypeConcordance.USAGE_SUMMARY + GenotypeConcordance.USAGE_DETAILS,
+        usageShort =  GenotypeConcordance.USAGE_SUMMARY,
         programGroup = VcfOrBcf.class
 )
 public class GenotypeConcordance extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Calculates the concordance between genotype data for samples in two different" +
+            "VCFs, your callset and a standard or 'truthset'.";
+    static final String USAGE_DETAILS = "Genotype concordance is the fraction of correct assigned genotypes divided by the" +
+            " total number of true-positive (TP) variant alleles.  Genotype concordance is calculated by dividing the number of validated " +
+            "genotypes in a call set by the total number of TP alleles.  Validated genotypes are variant " +
+            "genotypes that match a gold-standard data set.  The gold-standard data set can be any curated data set " +
+            "containing variant genotypes that have been cross-validated with multiple technologies " +
+            "e.g. Genome In A Bottle Consortium (GIAB) (https://sites.stanford.edu/abms/giab).  The TP variants are" +
+            " variant alleles that have been validated by a gold-standard database.  In contrast, a false-positive (FP)" +
+            " variant is a called variant allele that has not been validated by a gold-standard data base.  " +
+            "False-negative (FN) variants are variants incorrectly called as reference alleles and true negatives (TN)" +
+            " are simply validated reference allele calls."  +
+
+            "<h4>Usage example:</h4>" +
+            "<pre>" +
+            "java -jar picard.jar GenotypeConcordance \\<br />" +
+            "     -CALL_VCF=/Users/VCF1.vcf \\<br />" +
+            "     -O=/Users/GConcordance.vcf \\<br />" +
+            "     -TRUTH_VCF=/Users/GIAB.vcf \\<br />" +
+            "     -TRUTH_SAMPLE=sample#" +
+            "</pre>" +
+            "" +
+            "<h4>Output Metrics:</h4>" +
+            "Output metrics include GenotypeConcordanceContingencyMetrics, GenotypeConcordanceSummaryMetrics, and " +
+            "GenotypeConcordanceDetailMetrics.  For each set of metrics, the data is broken into separate sections for " +
+            "SNPs and indels.  Note that only SNP and INDEL variants are considered, MNP, Symbolic, and Mixed classes" +
+            " of variants are not included. <br /> <br /> +" +
+
+            "GenotypeConcordanceContingencyMetrics is a class that holds metrics " +
```

Comment:
Users don't want to hear about the "X is a class that holds metrics" stuff, that's developer-level information.